package.json

```diff
@@ -21,7 +21,7 @@
   "author": "the prebid.js contributors",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=4.0"
+    "node": ">=8.9.0"
```

Comment:
Could update https://github.com/prebid/Prebid.js/blob/master/.nvmrc as well to reflect this.

Reply:
@nanek Thanks for the suggestion.  I created a follow-up PR to make this change, #4162